pkgs/desktops/xfce/applications/xfce4-screensaver/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  mkXfceDerivation,
+  dbus-glib,
+  garcon,
+  glib,
+  gtk3,
+  libX11,
+  libXScrnSaver,
+  libXrandr,
+  libwnck3,
+  libxfce4ui,
+  libxklavier,
+  pam,
+  systemd,
+  xfconf,
+}:
+
+mkXfceDerivation {
+  category = "apps";
+  pname = "xfce4-screensaver";
+  version = "4.16.0";
+
+  sha256 = "1vblqhhzhv85yd5bz1xg14yli82ys5qrjdcabg3l53glbk61n99p";
+
+  buildInputs = [
+    dbus-glib
+    garcon
+    glib
+    gtk3
+    libX11
+    libXScrnSaver
+    libXrandr
+    libwnck3
+    libxfce4ui
+    libxklavier
+    pam
+    systemd
+    xfconf
+  ];
+
+  configureFlags = [ "--without-console-kit" ];
+
+  makeFlags = [ "DBUS_SESSION_SERVICE_DIR=$(out)/etc" ];
+
+  meta =  {
+    description = "Screensaver for Xfce";
```

Comment:
Is the rest of meta set automatically?

Reply:
I based this on other Xfce packages. Looking more closely, I see that homepage, license, and platform are set in pkgs/desktops/xfce/mkXfceDerivation.nix. However, it appears that file does not set a maintainer, and indeed most Xfce packages are currently maintainerless. What's the standard operating procedure, here? Should I add myself as a maintainer since I'm adding the package? I'm not sure yet how much or little time I'll have to contribute.

Reply:
> Should I add myself as a maintainer since I'm adding the package?

According to the contribution guide which you surely read completely you must do that.

Reply:
Done.